tests/integration/integration-tests.yml

```diff
@@ -1566,7 +1566,7 @@
           - test_five['rack']['tags'][0] == "Schnozzberry"
           - test_five['rack']['type'] == 100
           - test_five['rack']['u_height'] == 48
-          - test_five['rack']['width'] == "23"
```

Comment:
Do you mind adding or updating one of the tests to account for this bug? So have the rack created/updated with the width and then update it again to make sure it is still idempotent?